orion/sites/admin/src/achieve/achieveEdit.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+require('./achieveEdit.styl');
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Link } from 'react-router-dom';
+
+export class AchieveEditPage extends React.Component {
+constructor(props) {
+  super(props);
+  this.state = {
+    isEdit: false,
+    showDeleteModal: false,
+    showSaveModal: false,
+    inputYear: "",
+    inputMessage: "",
+  };
+}
+
+  render() {
+    const title = "Edit Achievement";
+    let deleteButton = <div></div>;
+    return (
+      <div id="view-achieve-edit">
+        <h2>{title}</h2>
+        <h4>Year</h4>
+        <input value={this.state.inputYear}
+                onChange={(e) => this.handleChange(e, "inputYear")}/>
```

Comment:
There should be a function called `handleChange`. Can you define the function? For now, it could just console.log something.